pkg/graveler/graveler.go

```diff
@@ -802,7 +802,7 @@ func NewKVGraveler(branchLocker BranchLocker, committedManager CommittedManager,
 }
 
 func (g *KVGraveler) GetRepository(ctx context.Context, repositoryID RepositoryID) (*Repository, error) {
-	return g.db.GetRepository(ctx, repositoryID)
+	return g.RefManager.GetRepository(ctx, repositoryID)
 }
 
 func (g *KVGraveler) CreateRepository(ctx context.Context, repositoryID RepositoryID, storageNamespace StorageNamespace, branchID BranchID) (*Repository, error) {
```

Comment:
why use the DB and not its kv ref manager?

Reply:
on all the other functions create, list repository

Reply:
Right. Done